src/Faithlife.Build/BuildRunner.cs

```diff
@@ -16,22 +16,26 @@ public static class BuildRunner
 	/// </summary>
 	/// <param name="args">The command-line arguments from <c>Main</c>.</param>
 	/// <param name="initialize">Called to initialize the build.</param>
+	/// <param name="settings">The build runner settings.</param>
 	/// <returns>The exit code for the build.</returns>
-	public static int Execute(string[] args, Action<BuildApp> initialize) =>
-		ExecuteAsync(args, initialize).GetAwaiter().GetResult();
+	public static int Execute(string[] args, Action<BuildApp> initialize, BuildRunnerSettings? settings = null) =>
```

Comment:
This is technically a binary breaking change. Would we prefer to add method overloads, or perhaps a method with a different name to allow void entry point execution?

Reply:
If the only use case here is unit testing, consider an internal overload.

Reply:
We should avoid the binary breaking change, yes.

I don't mind `BuildRunnerSettings` if we want to maintain both behaviors.